_Boards/Atmel/Board_Nano/MFBoards.h

```diff
@@ -36,30 +36,65 @@
 #define MF_DIGIN_MUX_SUPPORT 1
 #endif
 
+#ifndef MAX_OUTPUTS
 #define MAX_OUTPUTS         18
+#endif
+#ifndef MAX_BUTTONS
 #define MAX_BUTTONS         18
+#endif
+#ifndef MAX_LEDSEGMENTS
 #define MAX_LEDSEGMENTS     1
+#endif
+#ifndef MAX_ENCODERS
 #define MAX_ENCODERS        9
+#endif
+#ifndef MAX_STEPPERS
 #define MAX_STEPPERS        2
+#endif
+#ifndef MAX_MFSERVOS
 #define MAX_MFSERVOS        2
+#endif
+#ifndef MAX_MFLCD_I2C
 #define MAX_MFLCD_I2C       2
+#endif
+#ifndef MAX_ANALOG_INPUTS
 #define MAX_ANALOG_INPUTS   8
+#endif
+#ifndef MAX_OUTPUT_SHIFTERS
 #define MAX_OUTPUT_SHIFTERS 2
+#endif
+#ifndef MAX_INPUT_SHIFTERS
 #define MAX_INPUT_SHIFTERS  2
+#endif
+#ifndef MAX_DIGIN_MUX
 #define MAX_DIGIN_MUX       3
+#endif
 
 #define STEPS         64
 #define STEPPER_SPEED 400 // 300 already worked, 467, too?
 #define STEPPER_ACCEL 800
```

Comment:
I didn't wrap these because I don't think they are used? I need to open a separate issue for it, they only exist in the nano and uno MFBoard.h files.

Reply:
`define STEPS 64` is indeed not used, but `STEPPER_SPEED` and `STEPPER_ACCEL`  is used in `MFStepper.cpp`.

But we could think about to move it to the `MFStepper.h` file as it is more related to the stepper as to the boards and the speed and acceleration can be set by the connector now.